server/apps/archive/commands.py

```diff
@@ -28,26 +28,29 @@ def remove_expired_content(self):
         logger.info('Removing expired content')
         now = date_to_str(utcnow())
         items = self.get_expired_items(now)
+
         while items.count() > 0:
             for item in items:
                 logger.info('deleting {} expiry: {} now:{}'.format(item['_id'], item['expiry'], now))
                 superdesk.get_resource_service('archive').delete_action({'_id': str(item['_id'])})
```

Comment:
Are we checking if the spiked item is part of a package? Are we removing the referenced from the package?